.github/workflows/pre-release.yml

```diff
@@ -0,0 +1,81 @@
+# This workflow tests GMT installations on various platforms
+
+name: Test GMT releases
+
+# Run on master branch, PRs and scheduled times.
+on:
+  # push:
+  #   branches:
+  #   - master
+  pull_request:
+  # weekly schedule job
+  # schedule:
+  # - cron: '0 0 * * 0'
+
+env:
+  # The GMT version we're testing
+  GMT_VERSION: 6.4.0
+  # disable auto-display of figures
+  GMT_END_SHOW: off
+  # URL of the GMT Bundle
+  GMT_BUNDLE_URL: ftp://ftp.soest.hawaii.edu/release/gmt-6.4.0-darwin-x86_64.dmg
```

Comment:
@PaulWessel, can you comment with the correct path for this?

Reply:
I think it's `ftp.soest.hawaii.edu/pwessel/release/gmt-6.4.0-darwin-x86_64.dmg`

Reply:
Thanks